crates/gpui/src/platform/windows/platform.rs

```diff
@@ -26,7 +26,10 @@ use windows::{
     Win32::{
         Foundation::*,
         Graphics::{DirectComposition::*, Gdi::*},
-        System::{Com::*, Ole::*, Threading::*, Time::*},
+        Storage::FileSystem::{
+            GetFileVersionInfoSizeW, GetFileVersionInfoW, VerQueryValueW, VS_FIXEDFILEINFO,
```

Comment:
Let's use the * import variant, so we can cut down on spurious merge conflicts in the imports area.

Reply:
Done.